src/configs/default.json

```diff
@@ -61,7 +61,8 @@
                         "enabled": true,
                         "lifetime": true,
                         "today": true,
-                        "iv": true
+                        "iv": true,
+                        "limit": 20
```

Comment:
Maybe renaming this section to not "top20" would be better long term though.

Reply:
I agree, leaving as just `top` would be more clear.